pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -146,7 +148,8 @@ function validateParams(vmParams) {
             break;
         }
     } else {
-        validationFailed.source = _("Installation Source should not be empty");
+        if (vmParams.sourceType != DOWNLOAD_AN_OS)
```

Comment:
This can be shortened to `else if` but I really do not mind.

Reply:
Done